designer/client/src/components/graph/EspNode/element.ts

```diff
@@ -69,14 +69,30 @@ export function getStringWidth(str = "", pxPerChar = 8, padding = 7): number {
     return toString(str).length * pxPerChar + 2 * padding;
 }
 
+function getTestCounts(hasCounts: boolean, shortCounts: boolean, count: NodeCounts): string {
+    if (hasCounts) {
```

Comment:
Maybe it's my preferential but IMO it will be more readable

```
  if (!hasCounts) {
    return "";
  }

  if (shortCounts) {
    if (count && millify(count?.all)) {
      return count?.all?.toLocaleString();
    }
  } else if (count?.all?.toLocaleString()) {
    return count.all.toLocaleString() || "0";
  }
  
  return "?";
```

it's not blocking change, so resolve comment if you think we don't need it

Reply:
okay, maybe it's more readable :)